libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -644,6 +646,26 @@ void RC_Channel::do_aux_function(const aux_func_t ch_option, const aux_switch_po
         do_aux_function_camera_trigger(ch_flag);
         break;
 
+    case AUX_FUNC::FENCE: {
+        // enable or disable the fence
+        AC_Fence *fence = AP::fence();
+        if (fence != nullptr) {
+            AP_Logger *logger = AP_Logger::get_singleton();
+            if (ch_flag == HIGH) {
+                fence->enable(true);
+                if (logger != nullptr && logger->logging_enabled()) {
```

Comment:
The logging-enabled guard should be removed or we'll be changing Copter's behaviour.